JitsiConference.js

```diff
@@ -989,6 +989,32 @@ JitsiConference.prototype.replaceTrack = function(oldTrack, newTrack) {
         }, error => Promise.reject(new Error(error)));
 };
 
+/**
+ * Replaces the track at the lower level by going through the Jingle session
+ * and WebRTC peer connection. The track is replaced without the need for an
+ * offer/answer cycle.
+ * @param {JitsiLocalTrack} localTrack - the local track whose media stream has
+ * been updated.
+ */
+JitsiConference.prototype.replaceTrackWithoutOfferAnswer = function(localTrack) {
```

Comment:
Not a big fan of this new API. Why not make this an internal function which gets called by `JitsiConference.prototype.replaceTrack` if supported?